src/IslandoraUtils.php

```diff
@@ -230,7 +230,10 @@ public function getReferencingMedia($fid) {
     // Process field names, stripping off 'media.' and appending 'target_id'.
     $conditions = array_map(
       function ($field) {
-        return ltrim($field, 'media.') . '.target_id';
+        if (str_starts_with($field, 'media.')) {
+          $field = substr($field, strlen('media.'));
```

Comment:
we don't need to calculate a constant here

```suggestion
          $field = substr($field, 6);
```